unityfiles/Assets/Scripts/Game/MouseHandler.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MouseHandler : MonoBehaviour {
+    // Returns a vector from the position that our aim are based of, to the mouse position
+    public Vector3 MouseWorldPosition(GameObject aimDirectionPoint) {
+        return Camera.main.ScreenToWorldPoint(Input.mousePosition -
+                                              aimDirectionPoint.transform.position);
+    }
+    
+    // Returns an angle between two vectors in degrees
+    public float AngleBetweenTwoPoints(Vector3 vectorA, Vector3 vectorB) {
+        return Mathf.Atan2(vectorA.y - vectorB.y, vectorA.x - vectorB.x) * Mathf.Rad2Deg;
+    }
```

Comment:
Kanskje flytte denne ut av denne klassen, da den ikke har noe med musen å gjøre. Matte -> vector funksjon : lage hjelpe klasser for dette?

Reply:
Kanskje den skal hete noe som `GetGunAngle`?